README.md

```diff
@@ -1,6 +1,80 @@
-# TheBugTracker
-CAPSTONE BUG TRACKER PROJECT
+# Autobug
```

Comment:
Autobug Tracker?

Reply:
Oh yeah, your right. AutoBugTracker is what we called it with Daniel.